Flacky storybook tests

I've noticed our storybook tests fail every now and then with this error:

```
 Failed to capture stories.
{
  "name": "Error"
}
- Aborting build...
✔ Successfully aborted build.
Error: page.evaluate: TypeError: Cannot read properties of undefined (reading 'cacheAllCSFFiles')
    at eval (eval at evaluate (:226:30), <anonymous>:4:17)
    at UtilityScript.evaluate (<anonymous>:228:17)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
/home/runner/work/pixeleye/pixeleye/libs/ui:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @pixeleye/ui@0.0.3 pixeleye: `pnpm storybook & npx --yes pixeleye@latest storybook http://localhost:6006`
Exit status 1
```

--- a/integrations/storybook/src/capture.ts
+++ b/integrations/storybook/src/capture.ts
@@ -46,14 +46,24 @@ export async function captureStories({
 
   await Promise.race([
     page
-      .waitForFunction(() => (window as SBWindow).__STORYBOOK_CLIENT_API__, {
-        timeout: 60_000,
-      })
+      .waitForFunction(
+        () =>
+          (window as SBWindow).__STORYBOOK_CLIENT_API__._storyStore!
+            .cacheAllCSFFiles,
+        {
+          timeout: 60_000,
+        }
+      )
       .catch(() => {}),
     page
-      .waitForFunction(() => (window as SBWindow).__STORYBOOK_PREVIEW__, {
-        timeout: 60_000,
-      })
+      .waitForFunction(
+        () =>
+          (window as SBWindow).__STORYBOOK_PREVIEW__.storyStoreValue!
+            .cacheAllCSFFiles,
+        {
+          timeout: 60_000,
+        }
+      )
       .catch(() => {}),
   ]);
 

